tests/functional/Drag.ts

```diff
@@ -42,6 +42,9 @@ registerSuite('Drag', {
 		if (browserName === 'MicrosoftEdge') {
 			this.skip('For some reason, findById not working on Edge ATM.');
 		}
+		if (browserName === 'internet explorer') {
+			this.skip('Dragging is not working on Internet Explorer.');
```

Comment:
Note: I can load the functional test HTML page in IE11, click and drag, and the page functions as expected.  There seems to be some problem doing that through the remote interface.